.github/workflows/image_pr.yml

```diff
@@ -0,0 +1,38 @@
+name: imagePRlevel
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      commitSHA:
+        description: "pr commit SHA"
+        required: true
```

Comment:
do we have to provide sha manually or it can be set to the last commit?

Reply:
manually